examples/karaf-servlet-example/README.md

```diff
@@ -0,0 +1,62 @@
+# Apache Karaf Servlet examples
+
+## Abstract
+
+This example show how to register a servlet in the Karaf HTTP Service.
+
+It uses different approaches:
+
+1. Using `@WebServlet` annotation using the Karaf annotation scanner
+2. Using Karaf Servlet whiteboard pattern with blueprint
+3. Using Karaf Servlet whiteboard pattern with SCR
+
```

Comment:
We can add "4. Using the Karaf HTTP service." ?

Reply:
Good catch, thanks !